resources/windows/wix/directorylist.wxs.erb

```diff
@@ -1,9 +1,8 @@
 <?xml version='1.0' encoding='windows-1252'?>
 <Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
-
   <Fragment>
-    <DirectoryRef Id='TARGETDIR'>
-      <%= @platform.generate_wix_dirs(self.get_services) %>
+    <DirectoryRef Id="TARGETDIR">
+      <%= @platform.generate_wix_dir_structure(self.get_services, self) %>
```

Comment:
@McdonaldSeanp I think you would be better putting in the Wix directly for the top level structure and just using the ```get_services``` to list the service exclusions. My reasoning for this is:
1. Its much clearer to have the main structure written out in Wix rather than derived (the directory structure is complex enough without hiding it further :( )
2. What happens if the product has no services - will this code still run correctly.

Using the following segment would work (picked from change 47f522f57508fd6f2bf1458d89a5266afec1a0d2):
```
     <DirectoryRef Id='TARGETDIR'>
      <Directory Id="<%= settings[:base_dir] %>" >
        <Directory Id="<%= settings[:company_id] %>" Name="<%= settings[:company_name] %>">
          <Directory Id='INSTALLDIR' Name="<%= settings[:product_id] %>" />
        </Directory>
      </Directory>
      <Directory Id="CommonAppDataFolder" Name="CommonAppData">
        <Directory Id="<%= settings[:company_id] %>AppData" Name="<%= settings[:company_id] %>">
        </Directory>
      </Directory>
    </DirectoryRef>
```
This segment also corrects:
1. Correct company name for "Puppet Labs"
2. Uses Id only for the AppData and Program[64]Directory